sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -272,6 +350,36 @@ public virtual Response GetEnclaveQuotes(RequestContext context = null)
         /// <param name="content"> The content to send as the body of the request. </param>
         /// <param name="subLedgerId"> The sub-ledger id. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <example>
+        /// This sample shows how to call PostLedgerEntryAsync with required parameters and properties.
+        /// <code>
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("https://my-account-name.azure.com");
+        /// var client = new ConfidentialLedgerClient(endpoint, credential);
+        /// var data = new {
+        ///     contents = "LedgerEntryContents",
```

Comment:
Can you remind me how we decided to set the value of sample inputs?  I'm wondering if there's a way we can indicate it's a sample input and not a real input, let me follow up on this with the .NET architects.

Reply:
@annelo-msft Previously the proposal is as follows:
> Use `<outer>-<inner>-<propertyName>` to populate string parameter values.

In the [experimental example doc](https://review.docs.microsoft.com/en-us/dotnet/api/azure.analytics.purview.scanning.purviewscanclient.createorupdate?view=azure-dotnet-preview&branch=ShivangiTestBranchWithoutFiles#examples), the value is composed in the format `test-<propertyName>-<propertyName>xxxx`, like `test-scanResults-parentId`.

During the coding, I found that `Schema.DefaultValue` can be used to as mock values as well. So in current implementation, I'm using the `Schema.DefaultValue`.

Reply:
Where does Schema.DefaultValue come from?  Is this specified in the swagger or elsewhere?

Reply:
I think it's from M4. I'll investigate later.

Reply:
I've put the conclusion in the comments below.